robot/exercises/02_hardware_station_io.ipynb

```diff
@@ -148,10 +148,7 @@
    },
    "outputs": [],
    "source": [
-    "HTML(\n",
-    "    '<script src=\"https://unpkg.com/gojs/release/go.js\"></script>'\n",
-    "    + GenerateHtml(station)\n",
-    ")"
+    "HTML('<script src=\"https://unpkg.com/gojs/release/go.js\"></script>' + GenerateHtml(station))"
```

Comment:
can we try something like 
```
gojs = '<script src=\"https://unpkg.com/gojs/release/go.js\"></script>'
HTML(gojs + GenerateHtml(station))
```
to see if it works for the black formatter AND gradescope?

btw -- it's best to install the [pre-commit](https://pre-commit.com/) hooks in BOTH the solutions repo and the manipulation repo... it will format on commit (with all the right settings)

Reply:
Does that mean you tried it, and it works (for pre-commit AND gradescope)?